.github/workflows/bench_ec2_any.yml

```diff
@@ -27,22 +27,21 @@ on:
         default:
       archflags:
         description: Custom ARCH flags for compilation
-        default: -mcpu=cortex-a76 -march=armv8.2-a
+        default: ''
```

Comment:
The default was problematic: If you trigger a run where no cflags are needed and you thus clear them, the default would be picked. There's no distinction between "no cflags" and "empty cflags".